### ISSUE TITLE
bug: crash on init

### ISSUE THREAD

**Issue by bsherman** [ublue-os/fleek#53]
```
$ fleek init
 INFO  initializing fleek
Git Config - enter your email address: MYEMAILWASHERE
Git Config - enter your full name: MYNAMEWASHERE
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x9a8718]

goroutine 1 [running]:
github.com/ublue-os/fleek/nix.(*Flake).Init(0x0, 0x0?)
	/app/nix/flake.go:50 +0x38
github.com/ublue-os/fleek/cmd.initialize(0xc00028ac00?, {0xaca486?, 0x0?, 0x0?})
	/app/cmd/init.go:143 +0xb77
github.com/spf13/cobra.(*Command).execute(0xc00028ac00, {0x1037200, 0x0, 0x0})
	/root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0xc00028a000)
	/root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/root/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/vanilla-os/orchid/cmdr.(*App).Run(0xc00028a000?)
	/root/go/pkg/mod/github.com/vanilla-os/orchid@v0.4.0/cmdr/app.go:97 +0x78
main.main()
	/app/main.go:56 +0xfc5
```

This was on newest uBlue Kinoite (custom) as of March 22, 9pm EDT.

```
$ rpm-ostree status
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-unverified-registry:ghcr.io/bsherman/kinoite-custom:37
                   Digest: sha256:84bbd3540bf28c5c1909dfe4d524507f51610e9cd6aee5fd9a4b1d6e381d627c
                  Version: main (2023-03-23T01:11:06Z)
          LayeredPackages: langpacks-en
```

**Comment by bsherman** [comment]
I didn't think it would matter, but I also tested on a different machine that's Silverblue, not Kinoite. Same err.

**Comment by bketelsen** [comment]
fixed by #54
